include/grpcpp/impl/codegen/client_callback_impl.h

```diff
@@ -72,11 +72,14 @@ class CallbackUnaryCallImpl {
         grpc::internal::CallOpClientSendClose,
         grpc::internal::CallOpClientRecvStatus>;
 
-    auto* ops = new (::grpc::g_core_codegen_interface->grpc_call_arena_alloc(
-        call.call(), sizeof(FullCallOpSet))) FullCallOpSet;
-
-    auto* tag = new (::grpc::g_core_codegen_interface->grpc_call_arena_alloc(
-        call.call(), sizeof(grpc::internal::CallbackWithStatusTag)))
+    const size_t alloc_sz =
+        sizeof(FullCallOpSet) + sizeof(grpc::internal::CallbackWithStatusTag);
+    void* const alloced =
+        ::grpc::g_core_codegen_interface->grpc_call_arena_alloc(call.call(),
+                                                                alloc_sz);
+
+    auto* ops = new (alloced) FullCallOpSet;
+    auto* tag = new (alloced + sizeof(FullCallOpSet))
```

Comment:
nit: use `sizeof(*ops)` instead.

Reply:
Changed to use a struct to make things safer for alignment, so this should be moot now.